datascience/tables.py

```diff
@@ -1578,26 +1587,48 @@ def with_columns(self, *labels_and_values):
         return self
 
     def relabeled(self, label, new_label):
-        """Returns a table with label changed to new_label.
-
-        ``label`` and ``new_label`` may be single values or lists
-        specifying column labels to be changed and their new corresponding
-        labels.
+        """Returns a new table with ``label`` specifying column label(s)
+        replaced by corresponding ``new_label``.
 
         Args:
-            ``label`` (str or sequence of str): The label(s) of
+            ``label`` -- (str or array of str) The label(s) of
                 columns to be changed.
 
-            ``new_label`` (str or sequence of str): The new label(s) of
+            ``new_label`` -- (str or array of str): The new label(s) of
                 columns to be changed. Same number of elements as label.
 
-        >>> tiles = Table(['letter', 'count'])
-        >>> tiles = tiles.with_rows(
-        ...    make_array(make_array('c', 2), make_array('d', 4)))
+        Raises:
+            ``ValueError`` -- if ``label`` or ``new_label`` do not exist in
+                table, or if the two are not not of equal length. Also, raised
+                if ``label`` and/or ``new_label`` are not ``str``.
+
+        Returns:
+            New table with ``new_label`` in place of ``label``.
+
+        >>> tiles = Table().with_columns('letter', make_array('c', 'd'),
+        ...    'count', make_array(2, 4))
+        >>> tiles
+        letter | count
+        c      | 2
+        d      | 4
         >>> tiles.relabeled('count', 'number')
         letter | number
         c      | 2
         d      | 4
+        >>> tiles
+        letter | count
+        c      | 2
+        d      | 4
+        >>> tiles.relabeled(make_array('letter', 'count'),
+        ...   make_array('column1', 'column2'))
+        column1 | column2
+        c       | 2
+        d       | 4
+        >>> tiles.relabeled(make_array('letter', 'number'),
```

Comment:
Think you could add a comment here saying that this example shows that the original table is unmodified by `relabeled`?